### PR TITLE
Cleanup Wednesday #6

### PR DESCRIPTION
**Pull request by regiskuckaertz**
A few things indeed

- Goodbye legacy CAPI templates
- Remove artificial distinction between primary/secondary commercial modules
- Improve error reporting in commercial
- Harmonise bootstrap signatures
- Remove bonzo wrapper in creative templates
- Remove right-hand slot in paid for articles, Scala-style
- Remove unused flags in `commercial-features`
- Improve style

Things to do:
- [x] send reminder to merchandising teams
- [x] deactivate templates in DFP

### CODE REVIEW

**Comment by ScottPainterGNM** [comment]
On a Tuesday?! You've gone mad with power....

**Comment by regiskuckaertz** [comment]
giphy

**Comment by regiskuckaertz** [comment]
[Cleanup Wednesday 6](https://trello.com/c/p8oHdWuA/258-cleanup-wednesday-6)

File: static/src/javascripts-legacy/projects/commercial/modules/creatives/fabric-expanding-v1.js [hunk 1]
```diff
@@ -50,52 +50,49 @@ define([
...
         var viewportHeight = bonzo.viewport().height;
-        var adSlotTop = this.$adSlot.offset().top;
+        var adSlotTop = this.adSlot.getBoundingClientRect().top;
 
         var adHeight = (this.isClosed) ? this.closedHeight : this.openedHeight;
-        var inViewB = ((scrollY + viewportHeight) > adSlotTop);
-        var inViewT = ((scrollY - (adHeight * 2)) < adSlotTop + 20);
-        var topCusp = (inViewT &&
-        ((scrollY + (viewportHeight * 0.4) - adHeight) > adSlotTop)) ?
+        var inViewB = viewportHeight > adSlotTop;
```
Comment (regiskuckaertz) [inline static/src/javascripts-legacy/projects/commercial/modules/creatives/fabric-expanding-v1.js]: @katebee These expressions are just a means to detect whether the component is in the viewport. Now, I know that
```
bonzo(adSlot).offset().top ≡ window.pageYOffset + adSlot.getBoundingClientRect().top
```
and so all I do in these changes is drop the superfluous `window.pageYOffset`.

**Review by katebee — APPROVED** [review]
👍 :1st_place_medal:

**Pull request merged**